game/aamp_v2.ksy

```diff
@@ -0,0 +1,217 @@
+    - bbonectrl
+    - bchemical
+    - bchmres
+    - bdemo
+    - bdgnenv
+    - bdmgparam
+    - bdrop
+    - bgapkginfo
+    - bgapkglist
+    - bgenv
+    - bglght
+    - bgmsconf
+    - bgparamlist
+    - bgsdw
+    - bksky
+    - blifecondition
+    - blod
+    - bmodellist
+    - bmscdef
+    - bmscinfo
+    - bnetfp
+    - bphyscharcon
+    - bphyscontact
+    - bphysics
+    - bphyslayer
+    - bphysmaterial
+    - bphyssb
+    - bphyssubmat
+    - bptclconf
+    - brecipe
+    - brgbw
+    - brgcon
+    - brgconfig
+    - brgconfiglist
+    - bsfbt
+    - bsft
+    - bshop
+    - bumii
+    - bvege
+    - bactcapt
+  xref:
+    zeldamods: AAMP
+  endian: le
+seq:
+  - id: header
+    type: header
+    doc-ref: https://zeldamods.org/wiki/AAMP#Header
+  - id: parameter_lists
+    type: parameter_list
+    repeat: expr
+    repeat-expr: header.num_lists
+    doc-ref: https://zeldamods.org/wiki/AAMP#Parameter_list
+  - id: parameter_objects
+    type: parameter_object
+    repeat: expr
+    repeat-expr: header.num_objects
+    doc-ref: https://zeldamods.org/wiki/AAMP#Parameter_object
+  - id: parameters
+    type: parameter
+    repeat: expr
+    repeat-expr: header.num_parameters
+    doc-ref: https://zeldamods.org/wiki/AAMP#Parameter
+  - id: data_section
+    size: header.data_section_size
+    doc-ref: https://zeldamods.org/wiki/AAMP#Section_order
+  - id: string_section
+    size: header.string_section_size
+    doc-ref: https://zeldamods.org/wiki/AAMP#Section_order
+  - id: unknown_uint32_section
+    type: u4
+    repeat: expr
+    repeat-expr: header.num_unknown_uints
+    doc: Purpose is unknown. May be unused.
+    doc-ref: https://zeldamods.org/wiki/AAMP#Section_order
+types:
+  header:
+    seq:
+    - id: magic
+      contents: AAMP
+    - id: version
+      type: u4
+      doc: Should be "2"
+    - id: flags
+      type: u4
+      doc: |
+        TODO
+        Flags (LittleEndian 1 << 0, UTF8 1 << 1)
+        What does this mean?
+      doc-ref: https://zeldamods.org/wiki/AAMP#Header
+    - id: file_size
+      type: u4
+    - id: pio_verision
+      type: u4
+    - id: pio_offset
+      type: u4
+    - id: num_lists
+      type: u4
+    - id: num_objects
+      type: u4
+    - id: num_parameters
+      type: u4
+    - id: data_section_size
+      type: u4
+    - id: string_section_size
+      type: u4
+    - id: num_unknown_uints
+      size: 4
+      doc: number of uint32s after the string section
+    - id: pio_type
+      type: strz
+      size: 4
+      encoding: utf-8
+      doc: typically "xml"
+  parameter_list:
+    seq:
+      - id: name_crc32
+        type: u4
+      - id: child_lists
+        type: u4
+      - id: child_objects
+        type: u4
+    instances:
+      child_lists_offset:
+        value: child_lists & 16
+        doc: Offset to child lists, divided by 4 and relative to parameter list start
+      num_child_lists:
+        value: child_lists >> 16
+        doc: Number of child lists
```

Comment:
Alright. Fine. I kept them around only because I was sure there was some reason these numbers were grouped to gether in the reference material. Since everything you said about bit math went way over my head, I'm gonna choose the simpler option here.